src/gene/database/postgresql.py

```diff
@@ -834,14 +829,9 @@ def export_db(self, output_directory: Path) -> None:
         host = self.conn.info.host
         port = self.conn.info.port
         database_name = self.conn.info.dbname
```

Comment:
this line and the three lines above should be deleted since they're no longer used to form the shell command string

Reply:
no worries, done!